** After running the tests using py manage.py test, following were the result**

Found 9 test(s).
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
.Bob
........
----------------------------------------------------------------------Ran 9 tests in 18.733s

OK
Destroying test database for alias 'default'...
